source/fundamentals/write-data.txt

```diff
@@ -0,0 +1,147 @@
+.. _entity-framework-write-data:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: EF, EF Core, code example, write
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+{+framework-core+} allows you to work with data in your application without
```

Comment:
This constant is for "Entity Framework Core" (the framework itself) not for our provider. Since it's the product name I don't think having add "the" before would work here.